client/.gitignore

```diff
@@ -17,6 +17,7 @@
 
 # misc
 .DS_Store
+/server/.DS_Store
```

Comment:
I don't think we need to worry about server files in the client gitignore, this is for frontend stuff